include/swift/Serialization/ModuleFormat.h

```diff
@@ -570,7 +571,8 @@ namespace decls_block {
 
   using ParenTypeLayout = BCRecordLayout<
     PAREN_TYPE,
-    TypeIDField  // inner type
+    TypeIDField,                         // inner type
+    BCFixed<ParameterTypeFlags::NumBits> // vararg?, autoclosure?, escaping?
```

Comment:
Please don't use AST node info in ModuleFormat.h; it should be manually translated to "stable" encodings.